olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -96,8 +98,42 @@ class DATASERVICE_READ_API VersionedLayerClient final {
    * @param layer_id The layer ID of the versioned layer from which you want to
    * get data.
    * @param settings The `OlpClientSettings` instance.
+   *
+   * @deprecated Will be removed by 06.2020.
+   */
+  OLP_SDK_DEPRECATED(
+      "Use the ctor with the explicitly specified version. This ctor is "
+      "deprecated and will be removed "
+      "by 06.2020")
+  VersionedLayerClient(client::HRN catalog, std::string layer_id,
+                       client::OlpClientSettings settings);
+  /**
+   * @brief Creates the `VersionedLayerClient` instance with the specified
+   * catalog version.
+   *
+   * The instance of this client is locked to the specified catalog version
+   * passed to the constructor and can't be changed. This way we assure data
+   * consistency. Keep in mind that catalog version provided with requests like
+   * \ref DataRequest, \ref PartitionsRequest, and \ref PrefetchTilesRequest
+   * will be ignored.
```

Comment:
```suggestion
   * The instance of this client is locked to the specified catalog version
   * passed to the constructor and cannot be changed. This way, we assure data
   * consistency. Catalog versions provided with requests like
   * \ref DataRequest, \ref PartitionsRequest, and \ref PrefetchTilesRequest
   * are ignored.
```
Let's make the last sentence look like a general rule :)